pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -104,6 +104,7 @@ func init() {
 	cloudprovider.RegisterCloudProvider(ProviderName, func(config io.Reader) (cloudprovider.Interface, error) {
 		cfg, err := readConfig(config)
 		if err != nil {
+			glog.Errorf("Failed to initialize openstack cloudprovider: %s", err)
```

Comment:
Please change the cloudprovider.RegisterCloudProvider function to do this logging-- that way you'll fix it for all cloud providers, not just this one.